pkg/seedadmissioncontroller/webhooks/admission/extensionresources/admission.go

```diff
@@ -107,7 +107,7 @@ func New(logger logr.Logger, allowInvalidExtensionResources bool) *handler {
 			},
 		},
 
-		gvr("etcds"): {
+		gvr("etcds", gvrWithGroup(druidv1alpha1.GroupVersion.Group)): {
```

Comment:
You must set both the group and version to `druidv1alpha1.GroupVersion.Group` and `druidv1alpha1.GroupVersion.Version`. Yes, the version happens to be `v1alpha1` in both cases, but this is accidental and should not be relied upon.

Though I have no strong objections, I am not a big fan of the options approach in this case. I think using a function such as `gvrDruid` is simpler and therefore better.